src/modules/core/components/ProgressBar/ProgressBar.tsx

```diff
@@ -42,12 +42,13 @@ const ProgressBar = ({
 }: Props) => {
   const { formatMessage } = useIntl();
   const titleText = formatMessage(MSG.titleProgress, { value, max });
+  const shouldDisplayThreshold = !!(threshold && threshold > 0);
```

Comment:
`!!threshold` would have the same effect here. If `threshold` is falsy (e.g. `undefined` or `0`) it will return `false`, otherwise `true`.  Then you could remove this line and below do `!!threshold && (...)`

Reply:
I agree. I was keeping the threshold inside a variable because generally I like to use variable names to help document the code but in this case its simple enough that it isn't necessary. Thank you for the suggestion. I made the change already.